server-logic/src/main/java/com/olrox/chat/server/manager/UsersManagerFactory.java

```diff
@@ -1,3 +1,4 @@
+//what is this class needed
```

Comment:
The idea was that I can use this class to get any implementation of UsersManager(for example current works with UsersData class but another could work with database). Also we can rewrite this class to return the same instance of UsersManager instead of creating new each time. Now I can't decide if this class is necessary or not.